extensions/api/data-management/contractnegotiation/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/contractnegotiation/model/ContractNegotiationDto.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022 Florian Rusch
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributors:
+ *   Florian Rusch - Initial API and Implementation
+ */
+
+package org.eclipse.dataspaceconnector.api.datamanagement.contractnegotiation.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import org.eclipse.dataspaceconnector.spi.types.domain.contract.negotiation.ContractNegotiation.Type;
+
+@JsonDeserialize(builder = ContractNegotiationDto.Builder.class)
+public class ContractNegotiationDto {
+    private String id;
+    private String counterPartyAddress;
+    private String protocol;
```

Comment:
should be "ids-multipart" by default. Either initialize in constructor or as initializer.

Reply:
Fixed